$OpenBSD: patch-nDPI_Makefile_am,v 1.2 2020/10/02 18:47:14 sthen Exp $

not compatible with openbsd's nonstandard bpf_timeval - not installed in
the ntopng port anyway

Index: nDPI/Makefile.am
--- nDPI/Makefile.am.orig
+++ nDPI/Makefile.am
@@ -1,5 +1,5 @@
 ACLOCAL_AMFLAGS = -I m4
-SUBDIRS = src/lib example tests
+SUBDIRS = src/lib tests
 
 if BUILD_FUZZTARGETS
 SUBDIRS += fuzz
